locust/ramping.py

```diff
@@ -158,5 +159,6 @@ def ramp_down(clients, hatch_stride):
         locust_runner.hatch_rate = hatch_rate
     if start_count > 0:
         locust_runner.start_hatching(start_count, hatch_rate)
+    logger.info(locust_runner.running_type)
```

Comment:
I suggest to create better logger info message. maybe message like `Test Ramp Type : Auto` will be more readable